Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2003-10-14
Initial Package Version: 1.1.0
Origin: http://archive.linuxfromscratch.org/mail-archives/blfs-dev/2003-October/003975.html
        Posted by Michael Brömer
Description: Fixes compilation with jdk-1.4.2

$LastChangedBy: $
$Date: $

--- oo_1.1_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java.orig	2003-10-14 00:36:53.000000000 -0500
+++ oo_1.1_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java	2003-10-14 00:38:40.000000000 -0500
@@ -207,7 +207,7 @@
    index--;
   }
 
-  public org.openide.WizardDescriptor$Panel current()
+  public org.openide.WizardDescriptor.Panel current()
    {
      return panels[index];
    }
--- oo_1.1_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java.orig	2003-10-14 00:37:27.000000000 -0500
+++ oo_1.1_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java	2003-10-14 00:38:53.000000000 -0500
@@ -214,7 +214,7 @@
         index--;
     }
     
-    public org.openide.WizardDescriptor$Panel current() {
+    public org.openide.WizardDescriptor.Panel current() {
         return panels[index];
     }
     
--- oo_1.1_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java.orig	2003-10-14 00:37:55.000000000 -0500
+++ oo_1.1_src/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java	2003-10-14 00:39:05.000000000 -0500
@@ -200,7 +200,7 @@
         index--;
     }
     
-    public org.openide.WizardDescriptor$Panel current() {
+    public org.openide.WizardDescriptor.Panel current() {
         return panels[index];
     }
     
